homeassistant/components/tradfri/base_class.py

```diff
@@ -61,10 +61,13 @@ def device_info(self):
 
         return {
             "identifiers": {(TRADFRI_DOMAIN, self._device.id)},
-            "name": self._name,
             "manufacturer": info.manufacturer,
             "model": info.model_number,
+            "name": self._name,
+            "power_source": info.power_source_str,
```

Comment:
This isn't a valid key.

https://developers.home-assistant.io/docs/en/device_registry_index.html#device-properties